### ISSUE TITLE
We should embed the ELI5 Presto video from youtube on the website

### ISSUE THREAD

**Issue by psbell-meta** [prestodb/presto#19181]
[This video](https://www.youtube.com/watch?v=BGqIkiUtKHY) should be embedded near the top of https://prestodb.io/what-is-presto.html

**Comment by psbell-meta** [comment]
Will transfer issue to prestodb/prestodb.github.io once I have permissions

**Comment by wanglinsong** [comment]
This is done, please check https://prestodb.io/what-is-presto.html.

**Comment by wanglinsong** [comment]
@rohanpednekar please confirm and close this.

**Comment by tdcmeehan** [comment]
Looks like it's there, closing!
